src/backend/columnar/columnar_customscan.c

```diff
@@ -491,15 +515,324 @@ RelationIdGetNumberOfAttributes(Oid relationId)
 }
 
 
-static Path *
-CreateColumnarScanPath(PlannerInfo *root, RelOptInfo *rel, RangeTblEntry *rte)
+/*
+ * CheckPushdownClause tests to see if clause is a candidate for pushing down
+ * into the given rel (including join clauses). This test may not be exact in
+ * all cases; it's used to reduce the search space for parameterization.
+ *
+ * Note that we don't try to handle cases like "Var + ExtParam = 3". That
+ * would require going through eval_const_expression after parameter binding,
+ * and that doesn't seem worth the effort. Here we just look for "Var op Expr"
+ * or "Expr op Var", where Var references rel and Expr references other rels
+ * (or no rels at all).
+ */
+static bool
+CheckPushdownClause(RelOptInfo *rel, Expr *clause)
+{
+	if (IsA(clause, OpExpr))
+	{
```

Comment:
to be more explicit, maybe:
```suggestion
	if (!IsA(clause, OpExpr))
	{
		/* we only care about clauses like "Var op Expr" and "Expr op Var */
		return false;
	}
	
	......
	......		
```

Reply:
Explained in error message instead of comments.